src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/CombinationUtils.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public static class CombinationUtils
+    {
+        public static IEnumerable<IEnumerable<KeyValuePair<TKey, TValue>>> Enumerate<TKey, TValue>(IEnumerable<KeyValuePair<TKey, IEnumerable<TValue>>> possibleCombinations)
```

Comment:
>Enumerate [](start = 75, length = 9)

Please provide an example in XML docs like with letters and numbers or something.